packages/diagrams/frontend/sirius-components-diagrams/src/renderer/actions/useActions.tsx

```diff
@@ -0,0 +1,140 @@
+/*******************************************************************************
```

Comment:
- Please separate this into two different hooks `useActions` to retrieve actions and `invokeAction` to execute one.
- Please expose data and loading in the return value for `invokeActions`, we may need it in some use cases`
- Please expose loading for `useActions`

Reply:
done